lib/timeline/Timeline.js

```diff
@@ -120,15 +120,27 @@ function Timeline (container, items, groups, options) {
   this.itemsData = null;      // DataSet
   this.groupsData = null;     // DataSet
 
-  this.on('tap', function (event) {
```

Comment:
cool! But we should test it on mobile devices. It's possible that your screen emulates mouse events. I'll try to have a look on some devices.